Tasks/AzurePowerShellV4/azurepowershell.ts

```diff
@@ -1,8 +1,8 @@
 import fs = require('fs');
 import path = require('path');
 import os = require('os');
-import tl = require('azure-pipelines-task-lib/task');
-import tr = require('azure-pipelines-task-lib/toolrunner');
+import tl = require('vsts-task-lib/task');
```

Comment:
Please test in both Windows and Linux agents.

Reply:
Tested in both agents.